src/configuration/logger.py

```diff
@@ -0,0 +1,85 @@
+import os
```

Comment:
the name of the file and the name of the class should match i.e. logger.py = class Logger not loggerAdmin or vice versa and also why this inside the configuration module, this should be inside a utils or tools module but this is nothing related to a configuration